src/Security/Authorization.cs

```diff
@@ -140,7 +140,14 @@ public unsafe class Authorization : INativeObject, IDisposable {
 		[DllImport (Constants.SecurityLibrary)]
 		extern static int /* OSStatus = int */ AuthorizationCreate (AuthorizationItemSet *rights, AuthorizationItemSet *environment, AuthorizationFlags flags, out IntPtr auth);
 
-		[Deprecated (PlatformName.MacOSX, 10,7)]
+#if !NET
+		[Deprecated (PlatformName.MacOSX, 10, 7)]
```

Comment:
try not to modify the original attribute (extra space)

while we can review the PR without the extra noise using
https://github.com/xamarin/xamarin-macios/pull/12519/files?diff=unified&w=1

the changes makes it harder to find the original commit for the code (when the attribute was added)